test/playwright.spec.js

```diff
@@ -93,7 +93,7 @@ module.exports.describe = ({testRunner, product, playwrightPath}) => {
   describe('Browser', function() {
     beforeAll(async state => {
       state.browserApp = await playwright.launchBrowserApp(defaultBrowserOptions);
-      state.browser = await playwright.connect(state.browserApp.connectOptions());
+      state.browser = await playwright.connect({ wsEndpoint: state.browserApp.wsEndpoint() });
```

Comment:
So, we are now running our tests with websocket, while most users will probably use launch() and run with a pipe. That's unfortunate.

Reply:
oh, good catch. let me revert this bit.